utilities/test_suite/HOST/Tensor_host_audio.cpp

```diff
@@ -66,6 +66,10 @@ int main(int argc, char **argv)
     srcDescPtr = &srcDesc;
     dstDescPtr = &dstDesc;
 
+    // Set src/dst layout types in tensor descriptors
+    set_audio_descriptor_layout(srcDescPtr, dstDescPtr);
```

Comment:
I think for now
You can just the layout of srcDescPtr / dstDescPtr inside the cases directly, instead of having separate function